CHANGELOG.md

```diff
@@ -2,510 +2,321 @@
 
 ## Unreleased 3.0.0 (Breaking release)
 
-See the [versioning documentation for how to update this
-changelog](./docs/contributing/versioning.md#updating-changelog).
+## Breaking changes
+You must make the following changes when you migrate to this release, or your service may break.
 
-- Renamed css class names `js-character-count` and `js-header-toggle`
+### Update file paths, attributes and class names
+To make sure your project's files do not conflict with GOV.UK Frontend, we've moved our package files into a directory called `govuk`.
```

Comment:
I think we link to #1458 further down the changenote for those who need it. It's probably best to avoid 'further info' links in the actual migration instructions, because it tempts users to click through looking for more actual instructions.

Reply:
Yeah I was thinking that might be the case. Cool 👍